QuizApp/Scenes/LoginPage/Extensions/LoginVCExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoginVCExtension.swift
+//  QuizApp
+//
+//  Created by Tamuna Kakhidze on 12.11.24.
+//
+
+import UIKit
+
+// MARK: - LoginViewController constants extension
+extension LoginViewController {
+    enum Sizing {
+        static let blueBackgroundTopAnchor: CGFloat = 0
+        static let blueBackgroundLeadingAnchor: CGFloat = 0
+        static let blueBackgroundTrailingAnchor: CGFloat = 0
```

Comment:
if those paddings are zero you can remove it and just assign those constaints (where you are using it, probably in view controller) to be equal to .zero or even the superview